hermes/views.py

```diff
@@ -181,9 +180,137 @@ def get(self, request, *args, **kwargs):
+            targets: [
+                {
+                    name: <Target name>,
+                    ra: <Target ra in decimal or sexigesimal format>,
+                    dec: <Target dec in decimal or sexigesimal format>,
+                    ra_error: <Error of ra>,
+                    dec_error: <Error of dec>,
+                    ra_error_units: <Units for ra error>,
+                    dec_error_units: <Units for dec error>,
+                    pm_ra: <RA proper motion in arcsec/year>,
+                    pm_dec: <Dec proper motion in arcsec/year>,
+                    epoch: <Epoch in MJD>,
+                    orbital_elements: {
+                        epoch_of_elements: <>,
+                        orbinc: <>,
+                        longascnode: <>,
+                        argofperih: <>,
+                        eccentricity: <>,
+                        meandist: <>,
+                        meananom: <>,
+                        perihdist: <>,
+                        epochofperih: <>
+                    },
+                    discovery_info: {
+                        reporting_group: <>,
+                        discovery_source: <>,
+                        transient_type: <Type of source, one of PSN, nuc, PNV, AGN, or Other>,
+                        proprietary_period: <Duration that this discovery should be kept private>,
+                        proprietary_period_units: <Units for proprietary period, Days, seconds, Years>
+                    },
+                    redshift: <>,
+                    host_name: <Host galaxy name>,
+                    host_redshift: <>,
+                    aliases: [
+                        'alias1',
+                        'alias2',
+                        ...
+                    ],
+                    group_associations: <String of group associations for this target>
+                }
+            ],
+            photometry: [
+                {
+                    target_index: <Index of target list that this photometry relates to. Can be left out if there is only one target>,
+                    date_obs: <Date of the observation, in a parseable format or JD>,
+                    telescope: <Discovery telescope>,
+                    instrument: <Discovery instrument>,
+                    bandpass: <Wavelength band of the discovery observation>,
+                    brightness: <Brightness of the discovery>,
+                    brightness_error: <Brightness error of the discovery>,
+                    brightness_unit: <Brightness units for the discovery,
+                                      current supported values: [AB mag, Vega mag, mJy, erg / s / cm² / Å]>,
+                    new_discovery: <Boolean if this photometry is for a new discovery or not>
+                    exposure_time: <Exposure time in seconds for this photometry>,
+                    observer: <The entity that observed this photometry data>,
```

Comment:
I really like the use of the word "entity" here.
Could be a person, could be an organization, or it could be an AI or eldrich horror! Anything goes!